src/dfx/src/commands/canister/migrate_id.rs

```diff
@@ -49,34 +49,34 @@ pub async fn exec(
 
     // Get the canister IDs.
     let from_canister = opts.from_canister.as_str();
-    let to_canister = opts.rename_to.as_str();
+    let target_canister = opts.replace.as_str();
     let from_canister_id =
         Principal::from_text(from_canister).or_else(|_| canister_id_store.get(from_canister))?;
-    let to_canister_id =
-        Principal::from_text(to_canister).or_else(|_| canister_id_store.get(to_canister))?;
+    let target_canister_id = Principal::from_text(target_canister)
+        .or_else(|_| canister_id_store.get(target_canister))?;
 
-    if from_canister_id == to_canister_id {
-        bail!("From and rename_to canister IDs are the same");
+    if from_canister_id == target_canister_id {
+        bail!("From and target canister IDs are the same");
     }
 
     if !opts.yes {
         ask_for_consent(
             env,
             &format!(
-                "The from canister '{from_canister}' will be removed from its own subnet. Continue anyway?",
+                "The target canister '{target_canister}' will be removed from its own subnet. Continue anyway?",
```

Comment:
It is the `from_canister` that gets deleted! Moreover, I'd refer to it as the canister "to migrate".